static/css/introjs.scss

```diff
@@ -0,0 +1,347 @@
+.introjs-overlay {
```

Comment:
The static file story is getting messy. I changed the filetype to scss because plain css files are not copied over in the static dir. Is it ok if we open a follow-up bug to clean up the static directories? The one at server/src/static should be merged with the top level static dir, I think.